### PR TITLE
[new] Added foundation-datepicker@1.4.0

### PR DESCRIPTION
**Pull request by tjbp**
Repo can be found here: https://github.com/najlepsiwebdesigner/foundation-datepicker

### CODE REVIEW

File: ajax/libs/foundation-datepicker/package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    "zurb",
+    "foundation",
+    "datepicker"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/najlepsiwebdesigner/foundation-datepicker.git",
+    "basePath": "",
+    "files": [
+      "foundation-datepicker.min.js",
```
Comment (PeterDaveHello) [inline ajax/libs/foundation-datepicker/package.json]: They are placed in `css` and `js` folder and you missed it.

**Comment by tjbp** [comment]
@PeterDaveHello Whoops, fixed!

File: ajax/libs/foundation-datepicker/package.json (hunk 1 above)
Comment (tjbp) [inline ajax/libs/foundation-datepicker/package.json]: @PeterDaveHello Whoops, fixed!

**Comment by PeterDaveHello** [comment]
Would you please squash them into only one commit as our document says? Thanks.

**Comment by tjbp** [comment]
@PeterDaveHello righto, squashed

**Comment by PeterDaveHello** [comment]
Thanks

**Pull request merged**